src/main/kotlin/com/cognifide/gradle/aem/common/instance/service/osgi/ConfigurationState.kt

```diff
@@ -0,0 +1,23 @@
+package com.cognifide.gradle.aem.common.instance.service.osgi
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import com.fasterxml.jackson.annotation.JsonProperty
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+class ConfigurationState {
```

Comment:
other class have no `State` suffix. try to follow convention

I mean `ConfigurationState` -> `Configuration` / similar to `Bundle` , `Component` etc

Reply:
also be absolutely sure that none of configurations could have `null` somewhere.
I have such bug earlier. AEM reports suprisingly `null` in bundle `info` field as I remember (only once, temporarily and GAP failed then)

Reply:
Discussed offline - class name will stay as it is.